capstone-project/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -86,6 +99,31 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         }
     }
 
+    /** Verifies the idToken and returns the user ID if token is verified */
+    private static Optional<String> verifyToken(String idTokenString) throws IOException {
+        // If there's no user, the idTokenString received is an empty string
+        if (idTokenString.equals("")){
+            return Optional.empty();
+        }
+		try {
```

Comment:
fix indentations

Reply:
In the editor it looks fine, I don't know why it looks weird here 
![image](https://user-images.githubusercontent.com/54071800/96368114-b2d58a80-115a-11eb-8dfc-6d8079c1053c.png)